modules/videoio/include/opencv2/videoio/container_avi.private.hpp

```diff
@@ -0,0 +1,192 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef CONTAINER_AVI_HPP
+#define CONTAINER_AVI_HPP
+
+#ifndef __OPENCV_BUILD
+#  error this is a private header which should not be used from outside of the OpenCV library
+#endif
+
+#include "opencv2/core/cvdef.h"
+#include "opencv2/videoio/videoio_c.h"
+#include <deque>
+#include <stdint.h>
```

Comment:
This is C++11/C11 header: http://www.cplusplus.com/reference/cstdint/ like uint32_t and other types.